gammapy/irf/tests/test_energy_dispersion.py

```diff
@@ -93,30 +93,42 @@ def test_evaluation(self):
         assert_equal(actual, desired)
 
     def test_get_response(self):
-        # Get response
-        e_reco = EnergyBounds.equal_log_spacing(1 * u.GeV, 100 * u.TeV, 100)
-        response = self.edisp.get_response(1 * u.deg, 1 * u.TeV, e_reco)
-        actual = len(response)
-        desired = e_reco.nbins
-        assert_equal(actual, desired)
+        # Here we test get_response with an expected gaussian shape for edisp
+        from scipy.special import erf
+        # First we define a fake EnergyDispersion2D
```

Comment:
Suggest to split this whole part creating the test edisp object out into a separate method:
```
@staticmethod
def make_test_edisp2d():
    ...
    return edisp
```
Then if one ever needs to test it or if we want to call it from somewhere else, it's possible.
E.g.
```
from gammapy.irf.tests.test_energy_dispersion import TestEnergyDispersion2D
edisp = TestEnergyDispersion2D.make_test_edisp2d()
# play with it interactively or use it from other tests
```